schematic/schemas/commands.py

```diff
@@ -105,22 +115,38 @@ def convert(schema, data_model_labels, output_jsonld):
                 for w in war:
                     logger.warning(w)
 
+    if output_jsonld is None:
+        output_file_no_ext = re.sub("[.](jsonld|csv)$", "", schema)
+    else:
+        output_file_no_ext = re.sub("[.](jsonld|csv)$", "", output_jsonld)
+
+    logger.info(
+        "By default, the JSON-LD output will be stored alongside the first "
+        f"input CSV or JSON-LD file. In this case, it will appear here: '{output_jsonld}'. "
+        "You can use the `--output_jsonld` argument to specify another file path."
+    )
+
+    if output_type in ["graph", "all"]:
+        logger.info("Export graph to pickle if requested")
+        output_graph = output_file_no_ext + ".pickle"
+        try:
+            with open(output_graph, "wb") as file:
```

Comment:
Following convention can you move the pickle dump to `utils/schema_utils.py`, you can call it something like `export_graph` to mimic `export_schema` that is used to export the JSONLD.